gtk/tests_backend/screen.py

```diff
@@ -0,0 +1,25 @@
+import os
+
+import pytest
+from gi.repository import GdkX11
+
+from .probe import BaseProbe
+
+
+class ScreenProbe(BaseProbe):
+    def __init__(self, screen):
+        super().__init__()
+        self.screen = screen
+        self._impl = screen._impl
+        self.native = screen._impl.native
+        if os.environ.get("XDG_SESSION_TYPE", "").lower() == "x11":
```

Comment:
Again - an inconsistency between the probe and this test.

Reply:
Removed it.